transport/http/handler.go

```diff
@@ -82,6 +88,25 @@ func (h handler) callHandler(w http.ResponseWriter, req *http.Request) error {
 
 	ctx := context.Background()
 
+	// Extract opentracing etc baggage from headers
+	// Annotate the inbound context with a trace span
+	carrier := opentracing.HTTPHeadersCarrier(req.Header)
+	spanCtx, _ := h.Tracer.Extract(opentracing.HTTPHeaders, carrier)
+	span := h.Tracer.StartSpan(
+		treq.Procedure,
+		opentracing.StartTime(start),
+		opentracing.Tags{
+			"yarpc.caller":   treq.Caller,
+			"yarpc.service":  treq.Service,
+			"yarpc.encoding": treq.Encoding,
+		},
+		ext.RPCServerOption(spanCtx),
+	)
+	ext.PeerService.Set(span, treq.Caller)
+	// ext.HTTPUrl.Set(span, req.URL.String()) // XXX panics
+	// defer span.Finish() // XXX duplicate span submission
```

Comment:
panic due to `req.URL == nil` 

How do you determine duplicate submission, by observing the same span ID in two spans?  If you're capturing both client and server side spans, AND using Jaeger to test with, then this is expected behavior compatible with Zipkin, which represents an RPC call as a single span (we will be changing that in the new Jaeger data model).